selfdrive/test/process_replay/test_processes.py

```diff
@@ -154,38 +182,42 @@ def format_diff(results, ref_commit):
     untested = (set(interface_names) - set(excluded_interfaces)) - tested_cars
     assert len(untested) == 0, f"Cars missing routes: {str(untested)}"
 
-  results: Any = {}
+  pool = NestablePool(args.jobs)
+
+  lreaders: Any = {}
+  p1 = pool.map_async(get_logreader, [seg for car, seg in segments])
```

Comment:
we shouldn't need two pools here

Reply:
I think we do? We dont want to calculate lr each time, this pool runs less times.